### DISCUSSION TITLE
Fulfillment Inbound: Shipment Created Date

### DISCUSSION THREAD

**Discussion by ProNotion** [abuzuhri/Amazon-SP-API-CSharp#378] [Q&A]
In Seller Central we see the Shipping Queue and each shipment has a created date/time and last updated date/time. Where in the API can I find these values as they do not appear to be available as a part of the [InboundShipmentInfo](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v0-reference#inboundshipmentinfo)?

**Comment by RenzoF** [comment]
Based on the docs, it looks like they don't provide this via the API, they allow you to use it for queries but that's all

**Reply by ProNotion** [reply]
Thanks, I will raise an issue on their GitHub account as it would be very useful to have this.

**Reply by ProNotion** [reply]
There is no support for this currently which I'm very surprised at as the inbound shipments have very little context without a date.
